matthenge/weather-app
# Add styling

#### What does this PR do?

- Adds styling to the application components

#### Description of Task to be completed?
- Add styling to components
- Add styling to App.tsx

#### How should this be manually tested?
- Clone the repository
- Setup according to the README file
- Run the application
- The application should be styled

#### Any background context you want to provide?

#### What are the relevant issue Ticket?

#### Screenshots (if appropriate)

![Screenshot 2024-04-28 at 10 51 24](https://github.com/matthenge/weather-app/assets/22356695/af9356da-ff8d-44f4-90e0-e660fc74629e)

#### Lighthouse Screenshots (if appropriate)

#### Check List

#### Questions:
